workshops/portfolio/README.md

```diff
@@ -11,21 +11,70 @@ Your final design looks like this:
 
 > ![](img/final_1.png)
 
-Here is a link to a [live demo](http://output.jsbin.com/fugoki/2)
+Here is a link to a
+<a href="http://output.jsbin.com/fugoki/2" target="_blank">live demo</a>
 
-And here is the [final code](http://jsbin.com/fugoki/2/edit) for the live demo.
+And here is the
+<a href="http://jsbin.com/fugoki/2/edit" target="_blank">final code</a>
+for the live demo.
 
 To do this, you will be learning the basics of two languages: HTML and CSS.
 
 Every website that you have ever seen are written in these two languages.
 
-## Microsoft Word for Code
+We will be building this website on on an online code editor called JS Bin.
+
+## Creating a GitHub account
```

Comment:
I prefer "Creating" because "Create" sounds like an action that needs to be taken when you read it as opposed to a title of a section.